image/templates/helm/stackrox-central/templates/01-central-08-db-configmap.yaml

```diff
@@ -11,7 +11,11 @@ metadata:
     {{- include "srox.annotations" (list . "configmap" "central-db-config") | nindent 4 }}
 data:
   postgresql.conf: |
+    {{- if ._rox.central.db._postgresConfig -}}
```

Comment:
IF the existing way of completely overwriting the config map is present that will will take precedent of setting the values individually in the values file.  Does that sound like a reasonable approach to provide advanced users full control AND most importantly to provide backwards compatibility?